### ISSUE TITLE
Concurrent users

### ISSUE THREAD

**Issue by msaltieri** [Tychobra/polished#90]
Hello @merlinoa and congratulation for the amazing work.

I was wondering if you have already thought about a feature to prevent **two simultaneous login** with the same account. It would be great in order to enable a specific service to a limited set of users.

In my opinion the perfect behavior would be to force the logout of the previous session of the same user in the moment that a new session is requested. What do you think about it?

**Comment by merlinoa** [comment]
Thanks @msaltieri !  

I appreciate the suggestion; however, I don't think this would be expected behavior for most web applications.  e.g. I can sign into a single gmail account from multiple browser windows + tabs on a single computer, and I can also sign in from multiple devices at the same time.  simultaneous logins do no force other sessions with the same account to sign out.  I am pretty sure this is the default behavior for almost all web applications, so I am going to hold off on implementing this.  Let me know if I am missing something, and I would be happy to revisit.

**Comment by msaltieri** [comment]
Yes @merlinoa, I do agree with you. In almost all possible scenarios this functionality should not be activated.

However, in case you want to provide any type of SaaS, you probably wouldn't want the designated user to be able to give his access credentials to 10 other friends by paying only once for the service. A simple example would be Netflix.

However, I understand that it is not a priority. I think I will find no difficulty in inserting such a control directly inside the shiny server.

Thanks for your time!

**Comment by merlinoa** [comment]
Thanks @msaltieri 

Yes that is a good point and use case with Netflix.  But, yea, as you mention, I don't think the use case is going to occur often enough for us to make it a priority at this point.  I will keep it in mind for a future update.
